UndertaleModTool/Scripts/Community Scripts/TouchControlsEnabler.csx

```diff
@@ -100,11 +102,29 @@ ImportGMLFile(Path.Combine(dataPath, "gml_Object_obj_mobilecontrols_Step_0.gml")
 var mobileControls = Data.GameObjects.ByName("obj_mobilecontrols");
 mobileControls.Persistent = true;
 
-if (displayName == "deltarune chapter 1 & 2" || displayName == "deltarune chapter 1&2")
+var obj_time = Data.GameObjects.ByName("obj_time");
+if (obj_time is not null)
+{
+    Data.Code.ByName("gml_Object_obj_time_Create_0").AppendGML("instance_create(0, 0, obj_mobilecontrols);", Data);
+    return;
+}
```

Comment:
Deltarune has an obj_time as well as an obj_gamecontroller. (Undertale only has obj_time). I would advise you to put the obj_gamecontroller check first, or to check if adding the controls to obj_time works in Deltarune (in which case you could likely remove the obj_gamecontroller handling altogether).